docker/pulsar-all/pom.xml

```diff
@@ -125,6 +125,29 @@
               </execution>
             </executions>
           </plugin>
+          <plugin>
+            <groupId>pl.project13.maven</groupId>
+            <artifactId>git-commit-id-plugin</artifactId>
+            <version>${git-commit-id-plugin.version}</version>
+            <executions>
+              <execution>
+                <id>git-info</id>
+                <goals>
+                  <goal>revision</goal>
+                </goals>
+              </execution>
+            </executions>
+            <configuration>
+              <useNativeGit>true</useNativeGit>
+              <prefix>git</prefix>
+              <failOnNoGitDirectory>true</failOnNoGitDirectory>
+              <skipPoms>false</skipPoms>
+              <gitDescribe>
+                <skip>false</skip>
+                <always>false</always>
+              </gitDescribe>
+            </configuration>
+          </plugin>
```

Comment:
would it be possible to apply this in the parent `pom.xml`? This plugin will be useful in other contexts too.

Reply:
Good idea. I moved it to the parent `pom.xml` and skip it by default for other modules.